jenkins/catapult/jenkins/catapult-client-release-build.groovy

```diff
@@ -31,7 +31,7 @@ pipeline {
 	stages {
 		// stage('preliminary') {
 		//	 when {
-		//		 expression { is_public_build() && SHOULD_PUBLISH_BUILD_IMAGE.toBoolean() }
+		//		 expression { isPublicBuild() && SHOULD_PUBLISH_BUILD_IMAGE.toBoolean() }
 		//	 }
 
 		//	 steps {
```

Comment:
should you delete this commented out code instead?

Reply:
yea, will remove